packages/dev/test/RelayServerRequestsProfiling.test.ts

```diff
@@ -11,7 +11,7 @@ import { defaultEnvironment } from '@opengsn/common'
 
 contract('RelayServerRequestsProfiling', function (accounts) {
   const refreshStateTimeoutBlocks = 2
-  const callsPerStateRefresh = 13
+  const callsPerStateRefresh = 14
```

Comment:
we talked about decreasing this method count. why it keep rising ?

Reply:
Because we now read two different values from registrar, relayRegistrationMaxAge (which is configurable so can't be cached indefinitely) and relayInfo. I decreased it from 14 to 13 just a few commits  ago at #823 so it technically didn't increase for a long time.

Reply:
the point is that we have an event that happens once every 180 days (or at a rate we know, which is supposed to be very large), but we check it on every block...